File: accsaber-business/src/main/java/de/ixsen/accsaber/business/mapping/BusinessMappingComponent.java
package de.ixsen.accsaber.business.mapping;

import org.springframework.stereotype.Component;

@Component
public class BusinessMappingComponent {

    public ScoreSaberScoreMapper getScoreMapper() {
        return ScoreSaberScoreMapper.INSTANCE;
    }

    public ScoreSaberPlayerMapper getPlayerMapper() {
        return ScoreSaberPlayerMapper.INSTANCE;
    }
}

File: accsaber-business/src/main/kotlin/de/ixsen/accsaber/business/mapping/BusinessMappingComponent.kt
package de.ixsen.accsaber.business.mapping

import org.springframework.stereotype.Component

@Component
class BusinessMappingComponent {
    val scoreMapper: ScoreSaberScoreMapper
        get() = ScoreSaberScoreMapper.Companion.INSTANCE
    val playerMapper: ScoreSaberPlayerMapper
        get() = ScoreSaberPlayerMapper.Companion.INSTANCE
}